docs/backend/SYCL.md

```diff
@@ -300,6 +280,8 @@ For AMD GPUs we should expect at least one SYCL-HIP device [`hip:gpu`]:
 
 ### II. Build llama.cpp
 
+The SYCL backend depends on [oneMath](https://github.com/uxlfoundation/oneMath). By default it is automatically built along with the project. A specific build can be provided by setting the CMake flag `-DoneMath_DIR=/path/to/oneMath/install/lib/cmake/oneMath`.
```

Comment:
This description is not mandatory for Intel GPU.
Suggest move to NV/AMD GPU chatpers.

Reply:
Moved in https://github.com/ggml-org/llama.cpp/pull/12192/commits/06fe2ca7ed720521664edc504d2a0723a6a92039